pkg/costmodel/allocation.go

```diff
@@ -2724,7 +2708,6 @@ func (p Pod) AppendContainer(container string) {
 // TODO:CLEANUP add PersistentVolumeClaims field to type Allocation?
 type PVC struct {
 	Bytes     float64   `json:"bytes"`
-	Count     int       `json:"count"`
```

Comment:
count is replaced by the coefficient calculated with intervals and the number of allocations in a pod